### DISCUSSION TITLE
Cannot create a new print format

### DISCUSSION THREAD

**Discussion by amitsaxena01** [frappe/print_designer#126] [Q&A]
I have installed print designer on my erpnext instance. The version of ERPNext and Frappe is 15. However whenever I try to create a new format there is a blank canvas and it is not allowing me to add any elements. Can you help me to correct the issue. I am attaching the screenshot of the page that opens when I create a new print format using the designer
![Screenshot 2024-01-06 214853](https://github.com/frappe/print_designer/assets/10231694/5a9c2b76-4958-4a2f-b0ec-597914d4c366)

**Comment by maharshivpatel** [comment]
@amitsaxena01 can you please press f12 and share any error that is shown

### can you share your format like this :
You can paste the data here ( change any personal info )

https://github.com/frappe/print_designer/assets/39730881/b7b761c7-a2b1-43d4-a3b3-0dde25ba9699

**Reply by amitsaxena01** [reply]
Developer console shows no error. Screenshot is attached
![Screenshot 2024-01-08 202443](https://github.com/frappe/print_designer/assets/10231694/8e68e17d-ac10-4416-8865-05e7d8702d82)
Screen shot of the format are also attached along
![Screenshot 2024-01-08 203518](https://github.com/frappe/print_designer/assets/10231694/eed0c962-ff43-410b-bb04-edbb50e74808)
![Screenshot 2024-01-08 203534](https://github.com/frappe/print_designer/assets/10231694/0f40dc37-fe0e-42bd-951e-e723cdf56bd7)

**Reply by maharshivpatel** [reply]
@amitsaxena01  how are you hosting your site ? 
is it local setup ?

**Reply by amitsaxena01** [reply]
Yes, the erpnext instance is hosted on google cloud.

**Reply by maharshivpatel** [reply]
@amitsaxena01 did you figure this out ? have you restarted the bench ? are there any errors during installation ?

**Comment by gulito** [comment]
@amitsaxena01 did you already find a solution to your problem? I installed print designer today and have the same issue as you.
